tests/test_lib_caconnector.py

```diff
@@ -73,6 +78,14 @@
         "emailAddress=steve@openssl.org"
 
 
+@contextmanager
+def replace_stdin(target):
```

Comment:
Nifty :) Maybe one could also do this with ``mock.patch``, but I think this is nice already!